From 18b1875e527ccdd9e9b7b66dcfd98e6ae5ccd8cb Mon Sep 17 00:00:00 2001
From: Mark Ryan <mark.d.ryan@intel.com>
Date: Mon, 14 Jan 2019 11:11:27 +0100
Subject: [PATCH] Patch Eigen for TF 1.13

Two patches have been backported to the version of eigen used in TF1.13
that fix a build issue and fix a memory corruption error when using
complex numbers.

https://bitbucket.org/markdryan/eigen/commits/f3145b585b85659950ee7a8e3d1d2129e3120e27
https://bitbucket.org/markdryan/eigen/commits/49b5bfcfba835474762c424e53328cd1123243e2

Signed-off-by: Mark Ryan <mark.d.ryan@intel.com>
---
 tensorflow/workspace.bzl | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/tensorflow/workspace.bzl b/tensorflow/workspace.bzl
index aefab03..93fba1f 100755
--- a/tensorflow/workspace.bzl
+++ b/tensorflow/workspace.bzl
@@ -136,11 +136,11 @@ def tf_workspace(path_prefix = "", tf_repo_name = ""):
     tf_http_archive(
         name = "eigen_archive",
         build_file = clean_dep("//third_party:eigen.BUILD"),
-        sha256 = "753fbb58d0a49b6bcbcfb126ebfa2e21fc97f7471529ba835a096008ce588d8a",
-        strip_prefix = "eigen-eigen-9f48e814419e",
+        sha256 = "933189f7d35239a630074158d6e6cb8e1397289e8cbb66741ff007cb29b65e77",
+        strip_prefix = "eigen-git-mirror-tf1.13",
         urls = [
-            "https://mirror.bazel.build/bitbucket.org/eigen/eigen/get/9f48e814419e.tar.gz",
-            "https://bitbucket.org/eigen/eigen/get/9f48e814419e.tar.gz",
+            "https://github.com/markdryan/eigen-git-mirror/archive/tf1.13.tar.gz",
+            "https://github.com/markdryan/eigen-git-mirror/archive/tf1.13.tar.gz",
         ],
     )
 
-- 
2.7.4

